features/step_definitions/steps.rb

```diff
@@ -32,5 +32,9 @@ def path(pagename)
       new_user_registration_path
     when 'create menu'
       new_menu_path
+    when 'edit'
```

Comment:
same comment as before. Keep this in if it's generic and can be used for any resource.